api/v1alpha1/kustomization_types.go

```diff
@@ -28,7 +28,7 @@ type KustomizationSpec struct {
 	// +required
 	Path string `json:"path"`
 
-	// Label selector used for prune operations, e.g. env=staging.
+	// Label selector used for garbage collection.
 	// +kubebuilder:validation:Pattern="^.*=.*$"
 	// +optional
 	Prune string `json:"prune,omitempty"`
```

Comment:
Is a single label sufficient, or would a `[]string` be better?

Reply:
label selectors are not an array but an an expression e.g.:
`prune: "environment in (production),tier in (frontend)"`

Reply:
I've added the Kubernetes docs link to the readme.